endpoints/orders.yaml

```diff
@@ -66,13 +65,13 @@ paths:
               $ref: 'schemas/commonSchemas.yaml#/components/schemas/encodededPayloadWrapper'
       responses:
         '202':
-          description: Request processed successfully
+          description: Successful.
```

Comment:
We need to be more descriptive, for any response which references this schema (note line 72): "schemas/commonSchemas.yaml#/components/schemas/acceptedResponse" use text to indicate that the request was received by the SGS and another request will be sent by the SGS containing the async response payload. Something like this: "Request received, asynchronous response payload will be delivered to Counterparty endpoint createOrder. Counterparty endpoint URLs are set using the /v1/orders/subscriptions endpoint.